pvlib/test/test_irradiance.py

```diff
@@ -107,18 +108,25 @@ def test_get_extra_radiation(input, expected, method):
     assert_allclose(out, expected, atol=1)
 
 
-@requires_numba
-def test_get_extra_radiation_nrel_numba(times):
-    result = irradiance.get_extra_radiation(times, method='nrel', how='numba',
-                                            numthreads=8)
-    assert_allclose(result, [1322.332316, 1322.296282, 1322.261205, 1322.227091])
-
-
 def test_get_extra_radiation_epoch_year():
     out = irradiance.get_extra_radiation(doy, method='nrel', epoch_year=2012)
     assert_allclose(out, 1382.4926804890767, atol=0.1)
 
 
+@requires_numba
+def test_get_extra_radiation_nrel_numba(times):
+    with warnings.catch_warnings():
+        # don't warn on method reload or num threads
+        warnings.simplefilter("ignore")
+        result = irradiance.get_extra_radiation(times, method='nrel',
+                                                how='numba',
+                                                numthreads=4)
```

Comment:
since you're making formatting changes, possibly consider:
```python
result = irradiance.get_extra_radiation(
    times, method='nrel', how='numba', numthreads=4)
``` 
this is blacker code, tho not quite completely [Black](https://black.readthedocs.io/en/stable/)

Reply:
I agree this better formatting in this case. Overall, I prefer the grey zone.